### ISSUE TITLE
`load_utils` submodule is a duplicate of `io_utils` submodule

### ISSUE THREAD

**Issue by BrunoGrandePhD** [Sage-Bionetworks/schematic#284]
While browsing the code base, I noticed that the `load_utils` submodule is identical to the `io_utils` submodule. 

```
$ diff schematic/utils/io_utils.py schematic/utils/load_utils.py; echo $status
0
```
However, `load_utils` isn't imported anywhere in the repo. Searching for `(load|io)_utils` only turns up `io_utils` imports. I wanted to check if this is expected. If not, I can delete the duplicate `load_utils` in my next PR. 

```
schematic/schemas/explorer.py:
  14: from schematic.utils.io_utils import load_default, load_json, load_schemaorg

schematic/schemas/generator.py:
  10: from schematic.utils.io_utils import load_json

schematic/schemas/validator.py:
  4: from schematic.utils.io_utils import load_schemaorg, load_json, load_default

schematic/utils/__init__.py:
  5: from schematic.utils.io_utils import export_json, load_default, load_json, load_schemaorg

schematic/utils/validate_utils.py:
  2: from schematic.utils.io_utils import load_json
```

**Comment by milen-sage** [comment]
@sujaypatil96  I vaguely remember this resolved in one of the later PRs (but definitely possibility to have have imagined that :) )? Not super high priority, but something on our radar.

**Comment by sujaypatil96** [comment]
Yup! Made note of it, will resolve this is a `hotfix` PR sometime this week.

**Comment by sujaypatil96** [comment]
Addressed in PR #347.
